go.mod

```diff
@@ -101,7 +101,9 @@ require (
 
 replace (
 	github.com/go-yaml/yaml => github.com/go-yaml/yaml v2.2.8+incompatible
+	github.com/mattn/go-sqlite3 => github.com/mattn/go-sqlite3 v1.14.18
 	github.com/tektoncd/pipeline => github.com/tektoncd/pipeline v0.12.0
```

Comment:
Need to run `go mod tidy` to update go.sum

Reply:
I have run the `go mod tidy` but the issue is that library is not present in our go.sum. Instead it is being used by one of the libraries we use and that is the reason why we see this 😅 .

Reply:
since it is a dependency of another library, I guess we cannot do much unless they update it. So I though we can temporarily fix it using the `replace` 😅 . But if this is not the right way to do so, we should ignore the cve until there is an update from the other library's side. Please let me know what you think

Reply:
Yes I agree, but we still would have some updates in `go.sum`..right? Given we are introducing additional packages

Reply:
I tried to a `go mod tidy`, and there is no update to go.sum. I have no idea why 😅

Reply:
for example:
https://app.snyk.io/org/red-hat-openshift-data-science-rhods/project/570fd96f-09ad-488b-bb37-f822c34c973f#issue-SNYK-GOLANG-GITHUBCOMCONTAINERDCONTAINERD-6219724

This security issue says that it is caused because of 2 deps
 - github.com/operator-framework/operator-lifecycle-manager@v0.26.0
 - github.com/opendatahub-io/opendatahub-operator@v1.7.0
 
 I assumed that the transient deps will be added to go.sum but that is not the case. Yet the Snyk scan reports this. Now with the replaces fix, snyk says that the issue is fixed. I am confused.

Reply:
hum, when i used your go.mod from this PR , i have a lot cleaned up in go.sum

Reply:
🤔

Reply:
@AjayJagan You can try cleaning up the cache once
```
 go ​clean -modcache
```

Reply:
done still the same issue 😅